File: Cs/03_par_ou_impar.cs
using System;

public class ParOuImpar
{
    public static void Main(string[] args)
    {
        Console.Write("Digite um número inteiro: ");
        int numero = int.Parse(Console.ReadLine());

        if (numero % 2 == 0)
        {
            Console.WriteLine($"O número {numero} é PAR.");
        }
        else
        {
            Console.WriteLine($"O número {numero} é ÍMPAR.");
        }

        // Bônus: verificar de 1 a 10
        Console.WriteLine("\n--- Verificando de 1 a 10 ---");
        for (int i = 1; i <= 10; i++)
        {
            string tipo = (i % 2 == 0) ? "par" : "ímpar";
            Console.WriteLine($"{i} é {tipo}");
        }
    }
}

File: Java/03_par_ou_impar.java
import java.util.Scanner;

class par_ou_impar {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um número inteiro: ");
        int numero = scanner.nextInt();

        if (numero % 2 == 0) {
            System.out.println("O número " + numero + " é Par!");
        } else {
            System.out.println("O número " + numero + " é Ímpar!");
        }

        // Bônus: verificar de 1 a 10
        System.out.println("\n--- Verificando de 1 a 10 ---");
        for (int i = 1; i <= 10; i++) {
            String tipo = (i % 2 == 0) ? "par" : "ímpar";
            System.out.println(i + " é " + tipo);
        }

        scanner.close();
    }
}